test/unit/test-amp-img.js

```diff
@@ -377,6 +377,8 @@ describe('amp-img', () => {
   it('should propagate the object-fit attribute', () => {
     return getImg({
       src: '/examples/img/sample.jpg',
```

Comment:
Were these test breaking due to this change (if so why?)?

Reply:
I think the native Custom Elements implementations are able to fire `connectedCallback` much quicker than the `MutationObserver` the polyfill uses.